cli/lib/errors.js

```diff
@@ -9,13 +9,36 @@ const state = require('./tasks/state')
 
 const docsUrl = 'https://on.cypress.io'
 const requiredDependenciesUrl = `${docsUrl}/required-dependencies`
+const runDocumentationUrl = `${docsUrl}/command-line#cypress-run`
```

Comment:
This should be a new on link, since the hashes could change - this link would break.

Reply:
But the link would still go to `cypress run` CLI page, right. So it would not be that bad I feel

Reply:
Updated, adding redirects in https://github.com/cypress-io/cypress-services/pull/2714